src/Centreon/Application/Controller/PlatformTopologyController.php

```diff
@@ -179,4 +179,22 @@ public function getPlatformJsonGraph(): View
             return $this->view(['message' => $e->getMessage()], Response::HTTP_NOT_FOUND);
         }
     }
+
+    /**
+     * @param int $serverId
```

Comment:
IMHO Even if it is quite obvious what this function does a small doc about it could be great (_just a comment_)